src/main.rs

```diff
@@ -119,9 +125,15 @@ fn list_open_requests() {
     let mut tw = TabWriter::new(io::stdout()).padding(4);
     for mr in &mrs {
         if remote.has_useful_branch_names() {
-            writeln!(&mut tw, "{}\t{}\t{}", mr.id, mr.source_branch, mr.title).unwrap();
+            writeln!(
+                &mut tw,
+                "{}", format!("{}\t{}\t{}", mr.id, mr.source_branch, mr.title).green()
+            ).unwrap();
         } else {
-            writeln!(&mut tw, "{}\t{}", mr.id, mr.title).unwrap();
+            writeln!(
+                &mut tw,
+                "{}", format!("{}\t{}", mr.id, mr.title).green()
```

Comment:
Same color guidance as above: `green` and `uncolored`.

Reply:
Done.